test/email.test.js

```diff
@@ -0,0 +1,29 @@
+const { notifySignUp } = require('../app/helpers/mailer'),
+  request = require('supertest'),
+  app = require('../app.js'),
+  toEmail = 'email@wolox.com.ar';
+
+describe('Test for email-notifying users on sign up', () => {
+  beforeEach(() => {
+    notifySignUp.mockResolvedValueOnce(true);
+    notifySignUp.mockClear();
```

Comment:
To ensure the tests are unit tests, you could add `jest.clearAllMocks` to the `app.spec.js` before each, to make the mocks clear before each test. It would also save you from having to clear each mock individially.